pkg/services/pluginsintegration/clientmiddleware/tracing_middleware.go

```diff
@@ -135,3 +135,30 @@ func (m *TracingMiddleware) RunStream(ctx context.Context, req *backend.RunStrea
 	err = m.next.RunStream(ctx, req, sender)
 	return err
 }
+
+// ValidateAdmission implements backend.AdmissionHandler.
+func (m *TracingMiddleware) ValidateAdmission(ctx context.Context, req *backend.AdmissionRequest) (*backend.ValidationResponse, error) {
+	var err error
+	ctx, end := m.traceWrap(ctx, req.PluginContext, "ValidateAdmission")
```

Comment:
Not sure why we don't use the endpoint constants in this middleware.

```suggestion
	ctx, end := m.traceWrap(ctx, req.PluginContext, endpointValidateAdmission)
```